### PR TITLE
Info on integrating Ocular with other apps

### PR DESCRIPTION
**Pull request by NWeber-SL**
Want to make sure I get this info I have into the docs.

### CODE REVIEW

File: using-ocular/configure-extend/integrate-ocular.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+This article provides information on
+
+* [Encapsulating Results in a Script for Your CI and CD Pipeline](#encapsulating-results-in-a-script-for-your-ci-and-cd--pipeline)
+* [Debugging ShiftLeft Ocular Scripts with JDB](#debugging-shiftleft-ocular-scripts-with-jdb)
+
+## Encapsulating Results in a Script for Your CI and CD Pipeline
+
+Encapsulating ShiftLeft Ocular results in a script, and appending it to your CI/CD pipeline, enables you to measure improvements in results across releases.
+
+Deploy this command in your build system or any other system in your pipelie
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/configure-extend/integrate-ocular.md]:
```suggestion
Deploy this command in your build system or any other system in your pipeline
```

File: using-ocular/configure-extend/integrate-ocular.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+where `path` is the location and filename your application.
+
+The command:
+
+1. Executes ShiftLeft Ocular.
+2. Encapsulates the results in a script.
+3. Passes the script into a JAR file.
+4. Outputs the results.
+
+## Debugging ShiftLeft Ocular Scripts with JDB
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/configure-extend/integrate-ocular.md]: This section seems off-topic. I would remove this altogether or move to a different page.

File: using-ocular/configure-extend/integrate-ocular.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+where `path` is the location and filename your application.
+
+The command:
+
+1. Executes ShiftLeft Ocular.
+2. Encapsulates the results in a script.
+3. Passes the script into a JAR file.
+4. Outputs the results.
```
Comment (vlad-alexandru-ionescu) [inline using-ocular/configure-extend/integrate-ocular.md]: This information seems... strange. Who is your SME on this? I'm actually curious how this works.

File: using-ocular/configure-extend/integrate-ocular.md (hunk 1 above)
Comment (NWeber-SL) [reply using-ocular/configure-extend/integrate-ocular.md]: Fixed

File: using-ocular/configure-extend/integrate-ocular.md (hunk 2 above)
Comment (NWeber-SL) [reply using-ocular/configure-extend/integrate-ocular.md]: Done

File: using-ocular/configure-extend/integrate-ocular.md (hunk 3 above)
Comment (NWeber-SL) [reply using-ocular/configure-extend/integrate-ocular.md]: I think I got the info from a webinar

**Pull request closed without merging**